auto_nag/scripts/configs/tools.json

```diff
@@ -445,7 +445,7 @@
     "additional_receivers": ["jcristau@mozilla.com"],
     "component_exception": ["Testing::geckodriver"]
   },
-  "fuzzing_bisection_without_regressed_by": {
+  "bisection_without_regressed_by": {
     "additional_receivers": ["jkratzer@mozilla.com"]
```

Comment:
Did you check with Jason if he still wants to receive the emails?

Reply:
Jason replied that we can remove him from the list. Done in 55289b10b2ab6f4fe97039b6595cab784b609971.